Source/RuntimeAudioImporterEditor/Private/PreImportedSoundFactory.cpp

```diff
@@ -94,6 +95,9 @@ UObject* UPreImportedSoundFactory::FactoryCreateFile(UClass* InClass, UObject* I
 	
 	FMessageLog("Import").Error(ErrorText);
 	return nullptr;
+#else
+	UE_LOG(LogRuntimeAudioImporterEditor, Error, TEXT("File operation support disabled"));
+	return nullptr;
```

Comment:
I personally don't think we should have this on the editor side. Let’s skip this part please.